### PR TITLE
Button readme backtick

### PR DESCRIPTION
**Pull request by seangates**
## Description
Fixing README

## Screenshots
image

### CODE REVIEW

File: src/components/ebay-select/index.js [hunk 1]
```diff
@@ -120,11 +120,10 @@ function setSelectedOption(optionValue) {
         } else {
             option.selected = false;
         }
-        this.update();
         return option;
     });
-
     this.setState('options', options);
+    this.update();
```
Comment (senthilp) [inline src/components/ebay-select/index.js]: @seangates This change is not related to the button component. Is it still fine?

**Comment by seangates** [comment]
@senthilp Oh, I backed this one out. Please refer to #417 for the "true" PR.
